sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/example.env

```diff
@@ -0,0 +1,2 @@
+AZURE_FORM_RECOGNIZER_ENDPOINT=
```

Comment:
what's this file for?

Reply:
This is a dotenv file which is used to store configuration settings and sensitive information as environment variables. By keeping these values in a .env file, you can easily switch between different configurations without modifying your code.

https://stackoverflow.com/questions/68267862/what-is-an-env-or-dotenv-file-exactly

Reply:
You can check other packages, we usually don't include this file in samples. cc: @catalinaperalta

Reply:
Sure. Then let me remove it.

Reply:
Removed.

But pls check if you can use the dotenv file in the future. This will make the endpoint configuration much easier. :)
https://github.com/search?q=repo%3AAzure%2Fazure-sdk-for-python+load_dotenv&type=code